flink-table/flink-table-storage/src/main/java/org/apache/flink/table/storage/filestore/ValueType.java

```diff
@@ -18,31 +18,31 @@
 
 package org.apache.flink.table.storage.filestore;
 
-/** Value kind of a record. */
-public enum ValueKind {
+/** Value type of a record. */
+public enum ValueType {
```

Comment:
Do not use `Type`, there was a discussion before and the conclusion was that `type` was ambiguous.

Reply:
OK, let me revert this.